src/Context/EventContext.js

```diff
@@ -5,19 +5,12 @@ const EventContext = createContext()
 
 const EventProvider = props => {
   const [events, setEvents] = useState([])
-  const [searchedEvents, setSearchedEvents] = useState([])
-  const [filteredEvents, setFilteredEvents] = useState([])
-  const [category, setCategory] = useState('')
-  const [org, setOrg] = useState('')
+  const [category, setCategory] = useState('Any')
+  const [org, setOrg] = useState(null)
 
```

Comment:
have narrowed down considerably the number of data held in global context state. did not seem necessary to hold some of this in context when it is scoped to a single component